glue_qt/viewers/image/data_viewer.py

```diff
@@ -12,7 +12,6 @@
 # Import the mouse mode to make sure it gets registered
 from glue_qt.viewers.image.contrast_mouse_mode import ContrastBiasMode  # noqa
 from glue_qt.viewers.image.profile_viewer_tool import ProfileViewerTool  # noqa
-from glue_qt.viewers.image.pixel_selection_mode import PixelSelectionTool  # noqa
```

Comment:
It might make sense to preserve this import in a try...except if you are not able to import it from glue-core, to make sure we remain compatible with older glue-core versions.

Reply:
Where is the import happening now? I suppose it is not expected to pass without glue-viz/glue#2534, but right now it does not seem to raise on a failing `PixelSelectionTool ` import but on the missing `'image:point_selection'` key (your version in glue-core has `tool_id = 'image:selection_mode'`...)

Reply:
@astrofrog do you want to keep the local version in and just mark it deprecated until we up the glue-core deps to `glue>=1.23` or whatever it will be?